Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/UnityProjectInfo.cs

```diff
@@ -341,6 +358,51 @@ private List<PluginAssemblyInfo> ScanForPluginDLLs()
 
             return toReturn;
         }
+
+        private List<WinMDInfo> ScanForWinMDs()
```

Comment:
Could we combine this with ScanForDLLs to walk the file hierarchy once? You would probably have to do something with Directory.EnumerateFiles(Utilities.AssetPath, "*.*", SearchOption.AllDirectories); and loop over every file doing a Path.GetExtension on the file name.

Reply:
I combined things here, let me know if you have a different preferred search combination.